### PR TITLE
update icon: yii (original, plain, original-wordmark, plain-wordmark)

### LINKED ISSUE #1532: [BUG] Yii framework icons bugged and outdated
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### Bug description

There are problems with "yii" icons:
- `yii-original*` include raster images
- All versions are outdated, not latest version of official icons
- Color is wrong. Official color is `#40b3d8`, not `#0073bb`

### Possible fixes or solutions

Redo icons

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by cyberalien**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features
Yii framework icons redone, based on latest official logos from [yii framework website](https://www.yiiframework.com/logo).

**This PR closes #1532**
https://github.com/devicons/devicon/issues/1532

## Notes
Also fixes color in `devicon.json`

### CODE REVIEW

File: icons/yii/yii-plain-wordmark.svg [hunk 1]
```diff
@@ -1 +1 @@
...
....859-1.08-2.525-2.739-4.89-2.74-4.886-.003.011-1.495 8.047-4.175 13.86-.448.971-1.042 2.184-1.642 3.16-1.851 3.268-4.537 6.395-6.137 9.563-1.588 3.141-1.882 6.251-1.735 9.787.15 3.556.97 7.044 1.76 10.501 2.973-.642 5.56-1.739 7.799-3.142 5.89-3.694 9.45-9.6 10.462-15.965 0 0 .05-.258.071-.573.456-4.919.305-6.47.156-8.445Z"/><path d="M46.452 74.07c-.52-6.875-2.739-11.332-3.819-13.859-1.08-2.525-2.739-4.89-2.74-4.886v.002-.002l-.396-.59C30.809 42.542 14.149 36.453.072 42.096c-.678 8.549 3.285 23.256 17.743 27.358 5.845 1.798 10.525 1.332 16.261 2.892 0 0 5.83 2.032 9.223 5.08 1.525 1.37 3.053 3.173 2.977 5.318.483-5.105.328-6.666.176-8.672Z"/><path fill-rule="evenodd" d="M40.139 48.169c-2.055-6.01-1.184-10.037 2.574-15.641 1.793-2.674 4.887-5.776 7.6-7.528 10.943 6.858 14.99 19.63 10.965 31.493-2.929 8.632-5.677 12.251-12.625 21.132.81-9.511-2.538-16.105-5.825-23.432-.836-1.864-1.993-3.986-2.69-6.024Z" clip-rule="evenodd"/><path d="M46.276 82.743c.076-2.145-1.452-3.948-2.978-5.319-3.392-3.047-9.222-5.079-9.222-5.079.6-.976 1.194-2.189 1.642-3.16 2.68-5.813 4.172-13.849 4.175-13.86.001-.003 1.66 2.36 2.74 4.886 1.08 2.527 3.298 6.983 3.82 13.86.15 2.006.306 3.567-.178 8.672Z"/></svg>
```
Comment (Snailedlt) [inline icons/yii/yii-plain-wordmark.svg:1]: Paths need to be united

File: icons/yii/yii-plain.svg [hunk 2]
```diff
@@ -1 +1 @@
...
....018-2.478 13.342-6.923 22.983-.743 1.61-1.728 3.62-2.723 5.238-3.069 5.42-7.521 10.603-10.176 15.857-2.632 5.208-3.12 10.365-2.875 16.229.247 5.896 1.609 11.678 2.916 17.41 4.93-1.063 9.22-2.883 12.932-5.21 9.768-6.124 15.67-15.918 17.348-26.47 0 0 .082-.43.118-.952.757-8.155.506-10.729.258-14.002Z"/><path d="M89.023 81.366c-.864-11.402-4.54-18.792-6.331-22.98-1.79-4.189-4.542-8.109-4.544-8.103v.003-.004l-.657-.976c-14.405-20.221-42.03-30.317-65.371-20.96-1.124 14.174 5.447 38.56 29.42 45.363 9.691 2.981 17.452 2.208 26.963 4.794v.001s9.667 3.37 15.291 8.422c2.53 2.273 5.064 5.263 4.937 8.82.801-8.465.544-11.055.293-14.38Z"/><path fill-rule="evenodd" d="M78.555 38.417c-3.406-9.965-1.962-16.643 4.269-25.935C85.796 8.048 90.928 2.905 95.425 0c18.145 11.37 24.855 32.55 18.182 52.22-4.857 14.312-9.414 20.313-20.935 35.038 1.343-15.77-4.207-26.704-9.657-38.853-1.387-3.09-3.305-6.61-4.46-9.988Z" clip-rule="evenodd"/><path d="M88.731 95.746c.127-3.557-2.406-6.547-4.936-8.82-5.625-5.052-15.292-8.422-15.292-8.422.995-1.618 1.98-3.629 2.722-5.239 4.446-9.64 6.918-22.964 6.923-22.982.002-.006 2.754 3.914 4.544 8.102 1.79 4.189 5.468 11.579 6.332 22.98.251 3.326.508 5.916-.293 14.38Z"/></svg>
```
Comment (Snailedlt) [inline icons/yii/yii-plain.svg:1]: Paths need to be united (there should only be one path for line and plain versions)

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: icons/yii/yii-plain-wordmark.svg (hunk 1 above)
Comment (cyberalien) [reply icons/yii/yii-plain-wordmark.svg]: Fixed all icons. Also checked on icomoon to make sure they convert correctly to font.

**Review by Snailedlt — APPROVED** [review]

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this contribution! 🚀

**Pull request merged**